### ISSUE TITLE
Plasma Tech and Solar sats arnt calculated in "Ressource Settings"[BUG] 

### ISSUE THREAD

**Issue by akwarinho** [lanedirt/OGameX#247]
**Describe the bug**

**To Reproduce**
1. Go to Ressources'
2. Click on 'Ressource Settings'

**Expected behavior**
Building Solar Sats dont have an effect for me
same for plasma tech

**Comment by lanedirt** [comment]
Hi @akwarinho,

Thanks for the bug report. This behavior is expected due to the fact that the solar sats energy production and plasma technology bonus are not implemented yet. The backlog already contains issues to implement the logic for these two things.

These issues are related:
- #182 
- #170

**Comment by Geda173** [comment]
Hi @lanedirt,
is this topic still up to date? As far as I know, both issues have been resolved!

**Comment by lanedirt** [comment]
@Geda173, you're right, both of these should have been fixed with recent patches. 👍  Closing this issue.
